crates/viewer/re_selection_panel/src/selection_panel.rs

```diff
@@ -250,6 +250,24 @@ impl SelectionPanel {
                 }
 
                 if instance_path.is_all() {
+                    ui.list_item_flat_noninteractive(
+                        PropertyContent::new("Store entity").value_fn(|ui, _| {
```

Comment:
"Store entity" sounds like a command, and even if not it is a bit confusing ("Where else would the entity be?"). I suggest we just call it "Entity":

```suggestion
                        PropertyContent::new("Entity").value_fn(|ui, _| {
```